### PR TITLE
MDEXP-316 Fix error logs doesn`t contain item info

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
## Purpose
https://issues.folio.org/browse/MDEXP-316

## Approach
Since the structure of the JsonObject that goes into the mapping process has changed, we have to change the creation of AffectedRecords, so we added changes to this PR.

### CODE REVIEW

File: src/test/java/org/folio/service/logs/ErrorLogServiceUnitTest.java [hunk 1]
```diff
@@ -207,52 +205,14 @@ void saveWithAffectedRecord_shouldSaveWithHoldingsAndItemsInfo(VertxTestContext
...
-      assertTrue(ar.succeeded());
-      verify(errorLogDao).save(errorLogCaptor.capture(), eq(TENANT_ID));
-      ErrorLog errorLog = errorLogCaptor.getValue();
-      assertErrorLogWithHoldingsAndItems(errorLog, false);
-      context.completeNow();
-    }));
-  }
-
-  @Test
-  void saveWithAffectedRecord_shouldSaveWithHoldingsInfoWithoutItemIfHoldingIdIsEmpty(VertxTestContext context) {
```
Comment (KVupp) [inline src/test/java/org/folio/service/logs/ErrorLogServiceUnitTest.java:226]: WHy is this test case removed, it should still pass right?

**Review by KVupp — APPROVED** [review]
Just a minor comment regarding the removed test cases. Otherwise LGTM

File: src/test/java/org/folio/service/logs/ErrorLogServiceUnitTest.java (hunk 1 above)
Comment (andrii-novytskyi) [reply src/test/java/org/folio/service/logs/ErrorLogServiceUnitTest.java:226]: removed check when item holdingRecordId equals to holding, as we do it in fetchHoldingsAndItems() method while building json for mapping process, so we don`t need this test

**Comment by KVupp** [comment]
As part of this ticket can you also please increase the RMB version that was recommended to 31.1.5

**Review by IlliaDaliek — APPROVED** [review]

**Pull request merged**